app/wol.py

```diff
@@ -125,6 +129,16 @@ def load_computers():
 
 computers = load_computers()
 
+def send_raw_wol(mac_address, interface='eth0'):
```

Comment:
```suggestion
def send_raw_wol(mac_address, interface):
```
Default value already defined